### ISSUE TITLE
Not possible to translate the Assets Page title

### ISSUE THREAD

**Issue by espenlg** [statamic/cms#7098]
### Bug description

I've been working on the Norwegian translation, and I see the same in the Danish translation. When clicking on Assets -> Assets in the menu the title reads "Assets" over the list.

![Screenshot 2022-11-22 at 01 45 51](https://user-images.githubusercontent.com/1135679/203187959-18ab2ab7-9c51-43e3-9a6d-a76491ec2bfe.png)

And while I'm at it - the string "File" in xx.json, is this used in several places? I would like to call it "Name" instead as it is more like what I would expect if comparing to the MacOS finder or Windows Explorer. That is if it is only used above the file-/foldername list.

### How to reproduce

Try to use another language in CP and go to the Assets menu.

### Environment

```yaml
Environment
Application Name: Statamic
Laravel Version: 9.40.1
PHP Version: 8.0.23
Composer Version: 2.2.6
Environment: local
Debug Mode: ENABLED
URL: properti-master.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 3
Antlers: runtime
Version: 3.3.58 PRO

Statamic Addons
r8edge/imagemap: dev-espenlg/nov3
r8edge/welcome-widget: dev-main
statamic/eloquent-driver: 1.1.1
```

### Installation

Fresh statamic/statamic site via CLI

### Antlers Parser

runtime (new)

**Comment by jasonvarga** [comment]
That's because it's the title of the container.

It's the same issue as being able to translate other titles, like the title of a collection.

If you have a collection named `Articles` for example, you can add this to `lang/nb.json`

```json
{
  "Articles": "Artikler"
}
```

It will then show the translated collection name in the sidebar navigation. (I'm just noticing that only the sidebar is translated, not the title on the actual pages themselves.)

Assuming we implement the translation in the other spots, you could name your asset container something less generic then add it to the json file too.

```json
{
  "Articles": "Artikler",
  "Main Assets": "Viktigste Eiendeler"
}
```

![CleanShot 2022-11-22 at 09 55 57](https://user-images.githubusercontent.com/105211/203346137-4f142a66-9e8a-4652-a71a-4d47f61e68f8.png)

As for the "File" translation - it looks like the column header is currently the only place where that's used. But it might not be the only place forever.

**Comment by jasonvarga** [comment]
Keep it open, because translating the container title won't show the translation in all the places.

**Comment by espenlg** [comment]
The "File" translation is actually used in the Link fieldtype dropdown as well.
I would suggest the string in the column header is a unique string in the translation file. This applies for both English and Norwegian and would help having "Name" in English which after all is more correct than "File".

Example from the Norwegian translation. I've added ```"Asset": "Navn"``` to a local nb.json file.
![Screenshot 2022-11-24 at 14 55 37](https://user-images.githubusercontent.com/1135679/203801931-94ceb0cc-b5fe-425b-8454-f166d11a9c4e.png)

**Comment by ryanmitchell** [comment]
This should have been fixed as part of https://github.com/statamic/cms/pull/8664 
Is it still an issue on the most recent Statamic version?

**Comment by espenlg** [comment]
> This should have been fixed as part of #8664 Is it still an issue on the most recent Statamic version?
It's been a while so I haven't tried to be honest 🙈 I'll give it a go as soon as possible.

**Comment by espenlg** [comment]
> This should have been fixed as part of #8664 Is it still an issue on the most recent Statamic version?

Seems to be ok now 👍 One thing I noticed though, and not directly related to this, but if I translate "Assets" to lets say "Mediabank" it still has "Assets" as title when I click the menu-item "Mediabank".
![Screenshot 2023-11-19 at 19 08 19](https://github.com/statamic/cms/assets/1135679/6c35c2af-01d9-4dcc-950f-c3ad948ee6ae)
